core/src/test/java/com/homeaway/streamplatform/streamregistry/resource/RegionResourceIT.java

```diff
@@ -40,11 +40,11 @@ public void testGetRegions(){
 
         Collection<Hint> hints = (Collection<Hint>)response.getEntity();
 
-        Assert.assertEquals(new HashSet<>(Arrays.asList(US_EAST_REGION)),
+        Assert.assertEquals(new HashSet<>(Collections.singletonList(US_EAST_REGION)),
```

Comment:
Does this change matter ?

Reply:
This is not a significant change and was just recommended by IntelliJ for performance (the specific reason being that since it is a single entry in a list, it is better to use a singletonList for that purpose). Can revert if you want. :)

Reply:
Ahh... single entry in a list. Learned something! Thanks for sharing. Leave it... don't revert.